lib/tasks_private/spec.rake

```diff
@@ -7,5 +7,5 @@ desc "Run all specs"
 RSpec::Core::RakeTask.new(:spec => ["app:test:initialize", "app:evm:compile_sti_loader"]) do |t|
   spec_dir = File.expand_path("../../spec", __dir__)
   EvmTestHelper.init_rspec_task(t, ['--require', File.join(spec_dir, 'spec_helper')])
-  t.pattern = FileList[spec_dir + '/**/*_spec.rb'].exclude(spec_dir + '/manageiq/**/*_spec.rb')
+  t.pattern = FileList[spec_dir + '/**/*_spec.rb']
```

Comment:
This will run all the tests if specs/manageiq is a symlink (instead of a clone) see [bin/update](https://github.com/cfcosta/manageiq-providers-hawkular/blob/eff59769d1babb023ebf41ce86f0cb6f79bf1df4/bin/update#L6-L7).
We should verify if we are dealing with a symlink and update the `pattern`